boefjes/boefjes/katalogus/routers/plugins.py

```diff
@@ -39,7 +39,13 @@ def list_plugins(
     plugin_service: PluginService = Depends(get_plugin_service),
 ) -> list[PluginType]:
     with plugin_service as p:
-        plugins = p.get_all(organisation_id)
+        if filter_params.ids is not None:
```

Comment:
This would suffice, since you're doing a `None` check and and emptiness check
```suggestion
        if filter_params.ids:
```